src/io.c

```diff
@@ -1007,6 +1007,16 @@ UInt OpenOutputStream (
     return 1;
 }
 
+Obj FuncOPEN_OUTPUT_STREAM (
+    Obj                      self,
+    Obj                      stream )
+{
+    if(OpenOutputStream(stream)){
```

Comment:
Please use `git clang-format` on any newly added kernel code.

Reply:
This function kinda re-introduces "SET_OUTPUT" that @fingolfin removed a couple of days ago. I am not sure this is a good idea?